.github/workflows/manual-release.yml

```diff
@@ -0,0 +1,20 @@
+name: Manual Release
+# Triggers a merge from main->release, which will then trigger a release
+# from the release branch.
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  sync-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+
+      - name: Merge main -> release
+        uses: devmasx/merge-branch@master
```

Comment:
I wonder if this works if the branch is protected? I thought there is protection against directly merging without a PR.

Reply:
Check out the branch protection for that branch and lmk what you think.  I set it up so that only repo admins can push to it, but it does not require a PR.